js/models/settings/Settings.js

```diff
@@ -67,4 +67,17 @@ export default class extends BaseModel {
 
     return super.sync(method, model, options);
   }
+
+  parse(response = {}) {
+    if (Array.isArray(response.blockedNodes)) {
+      // de-dupe
+      response.blockedNodes = Array.from(new Set(response.blockedNodes));
```

Comment:
Since the items in a Set must be unique, it's a really cool way to de-dupe an array.